async-nats/src/lib.rs

```diff
@@ -630,7 +650,11 @@ impl ConnectionHandler {
                     };
                     debug!("reconnected");
                 }
-                ExitReason::Closed => break,
+                ExitReason::Closed => {
+                    //
+                    self.connector.events_tx.try_send(Event::Closed).ok();
```

Comment:
Similar to the other comment, this is more-or-less the last thing we'd do during shutdown so it's probably fine to ignore errors.

Reply:
It is, however please either remove the `//` or add a comment there that explains why its ok.